parser/fuzz/fuzz_targets/structured.rs

```diff
@@ -0,0 +1,46 @@
+#![no_main]
+
+mod common {
+	include!(concat!(env!("OUT_DIR"), "/common.rs")); // from build.rs
+}
+
+use ezno_parser::{ASTNode, Module, ParseOutput, SourceId, ToStringSettingsAndData};
+use libfuzzer_sys::{fuzz_target, Corpus};
+use pretty_assertions::assert_eq;
+
+fn do_fuzz(data: common::FuzzSource) -> Corpus {
+	let input = data.source;
```

Comment:
This should be valid JS code in this instance, so we should have fewer `Corpus:Reject`s / failed parses